### PR TITLE
Create poll

### PR DESCRIPTION
**Pull request by elsmr**
Create poll now works

### CODE REVIEW

**Review by hannnegoossens — APPROVED** [review]
Well done (y)

**Comment by Haroenv** [comment]
Aren't we supposed to review?
You were pair programming, that doesn't count

File: src/main/resources/templates/index.twig [hunk 1]
```diff
@@ -11,4 +11,11 @@
     </li>
     {% endfor %}
 </ul>
-{% endblock %}
\ No newline at end of file
+<form method="POST">
```
Comment (Haroenv) [inline src/main/resources/templates/index.twig:14]: add `action="/polls/create"` or something, and IMO it should be in that controller, not in the home controller

**Review by Haroenv — COMMENTED** [review]
Overall good changes though

File: src/main/resources/templates/index.twig (hunk 1 above)
Comment (Haroenv) [reply src/main/resources/templates/index.twig:14]: If the controller isn't going to change, then `action="/"`

Comment (Haroenv) [reply src/main/resources/templates/index.twig:14]: Also I'd use `post` instead of `POST` ([source](http://stackoverflow.com/a/4106613/3185307))

Comment (elsmr) [reply src/main/resources/templates/index.twig:14]: I'll change it

**Pull request merged**